docusaurus/docs/engineering/boba-backend/APIs/intro.md

```diff
@@ -4,23 +4,23 @@ sidebar_position: 1
 
 # Introduction to APIs
 
-APIs allow applications (clients) to interact with another application (server) without unfettered access to the underlying data, or to the whole set of operations that can be executed on it. In short, an API is a contract defined by the server that establishes a common language that clients can use to communicate with it. Most operations allowed by APIs boil down to fetching, storing or updating data.
+APIs allow different applications to talk to each other without giving them complete access to all the data or operations of the other application. In simple terms, an API is a contract that defines a common language so one application can communicate with another. Most API operations involve fetching, storing, or updating data.
 
 :::note
 
 You can explore the full BobaBackend API on the [REST API documentation page](/docs/engineering/rest-api/).
 
 :::
 
-While there are no enforced rules about how APIs should be defined, BobaBackend's API implements [a special set of API guidelines](https://www.redhat.com/en/topics/api/what-is-a-rest-api) called REST. Among the advantages of adhering to REST principles, is that REST APIs are easier to reason about, define, and scale. You can read about the REST principles followed by BobaBackend in the [API guidelines](./API-guidelines) page.
+While there are no enforced rules about how APIs should be defined, BobaBackend's API implements [a special set of API guidelines](https://www.redhat.com/en/topics/api/what-is-a-rest-api) called REST. Following REST principles has several advantages, including easier understanding, definition, and scalability of APIs. You can find more details about the REST principles followed by BobaBackend in the [API guidelines](./API-guidelines) page.
 
 ## How a REST API works
 
-REST APIs rely on the HTTP protocol, the same browsers use to navigate most webpages. In particular, a REST API defines a set of `endpoints` (URLs) that the client can access to execute operations. A REST API also defines the semantic of the data the client will send to describe the operation parameters, as well as the one of the data it will receive in response.
+REST APIs use the HTTP protocol, which is the same protocol that web browsers use to navigate websites. Specifically, a REST API defines a set of URLs called 'endpoints' that the client (your application) can access to perform operations. A REST API also specifies the format of the data the client needs to send for the operation and the format of the data it will receive in response.
```

Comment:
Somewhere in this paragraph, I'd like to see a mention of the thing that was tripping me up: that endpoints all share the same root URL and that root URL is the _server address_ specifically. I have no idea why it wasn't super clear to me from the beginning, but since it wasn't, I'm going to guess that a lot of people without backend experience won't get that upfront.